Altis_Life.Altis/stringtable.xml

```diff
@@ -3829,7 +3829,7 @@
         <Key ID="STR_NOTF_ActionDelay">
             <Original>You're doing it too fast!</Original>
             <Czech>Děláte to příliš rychle!</Czech>
-            <French>Moins vite !</French>
+            <French>Moins vite entre chaques actions!</French>
```

Comment:
We need a space before the "!", and I'd put it in singular. 
"Moins vite entre chaque action !"